src/main/java/com/google/maps/DirectionsException.java

```diff
@@ -0,0 +1,21 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.maps;
+
+public class DirectionsException extends Exception {
+  public DirectionsException(String message) {
+    super(message);
+  }
+}
```

Comment:
Also create a constructor `DirectionsException(String message, Throwable cause)` https://docs.oracle.com/javase/7/docs/api/java/lang/Exception.html#Exception(java.lang.String,%20java.lang.Throwable)

Reply:
Created, thanks! I assume this is to change `throw new DirectionsException("Failed to get directions " + e)` to `throw new DirectionsException("Failed to get directions", e)` is that right?

Reply:
Created, thanks! I assume this is to change `DirectionsException(""Failed to get directions " + e)` to `DirectionsException(""Failed to get directions", e)` is that right?

Reply:
Sorry - I just read the comment below, disregard my question.

Reply:
`RESOLVED`